src/main/java/billgates/view/gui/TopMenuBar.java

```diff
@@ -114,4 +146,42 @@ public void loadFile(File file) {
         // CONTINUE!!!
         System.out.println("loaded");
     }
+
+    //Function to show a description of the function of this app
+    private void showFunctions(){
+        JOptionPane.showMessageDialog(null, "<html>Username, Password, and Sign In:" +
+                        "<br>You will need to enter a username and password to sign in. " +
```

Comment:
Also, if you want to use the <html> tag, you can actually try using HTML formatting. 

For example `<h1>I am header 1</h1>` renders into <h1>I am header 1</h1>

Similarly, `<h2>I am header 2</h2>` also renders into something that could be a title. You can play around with the different header size and format these messages a little bit better, so they look nicer :3

Reply:
<img width="456" alt="image" src="https://user-images.githubusercontent.com/8016180/205518797-538b039d-012a-4983-8bae-fcff81cc5f3e.png">

After messing around a little bit, it might look like this